src/BaGet.Services.Mirror/Services/FakeMirrorService.cs

```diff
@@ -1,10 +1,20 @@
-﻿using System.Threading.Tasks;
+﻿using Microsoft.Extensions.Logging;
 using NuGet.Versioning;
+using System;
+using System.Threading.Tasks;
 
 namespace BaGet.Services.Mirror
 {
     public class FakeMirrorService : IMirrorService
     {
+
+        private readonly ILogger<MirrorService> _logger;
+
+        public FakeMirrorService(
+            ILogger<MirrorService> logger)
+        {
+            _logger = logger ?? throw new ArgumentNullException(nameof(logger));

```

Comment:
What's this logger for?

Reply:
good question:
not commited the one dummy test message to the logger, that i have used to verify that the output goes to ITestoutputhelper

Reply:
should i
a) remove the logger argument
b) add a logger output to the Fakeserver

it is not a big difference ...if i hear nothing from your side i do 
b) 
as next fix for this PR